### DISCUSSION TITLE
OAuth2 android redirect URLs support

### DISCUSSION THREAD

**Discussion by nktfh100** [pocketbase/pocketbase#1402] [Q&A] [answered]
Hello,

I'm trying to setup Google authentication with pocketbase and react native (with expo-auth-session). However, I'm running into an issue with the redirect URL for OAuth2 on Android.

The redirect URL looks like "com.example.app:/oauthredirect", which gives the validation error "must be a valid URL".

**Comment by ganigeorgiev** [comment, marked as answer]
You can check this discussion https://github.com/pocketbase/pocketbase/discussions/442#discussioncomment-4160820.

Since Google doesn't allow custom url schemes for the OAuth2 deeplink redirect, you can try using a "dummy" url as a redirect address, eg. "https://myapp.com/android-redirect" and capture it in your app.
There is one caveat though and as mentioned in https://github.com/pocketbase/pocketbase/discussions/442#discussioncomment-4172024 you'll have to prove that you own the domain used in the intent filter - https://developer.android.com/training/app-links/verify-android-applinks.

**Comment by nktfh100** [comment]
I see, thanks!
